docs/sp-add-ins/use-the-experimental-desktop-list-view-widget-in-sharepoint-add-ins.md

```diff
@@ -334,7 +334,7 @@ new Office.Controls.ListView(
 
 <br/>
 
-You can use the experimental Desktop List View widget to display data in SharePoint lists. The widget displays data in read-only mode. Please provide ideas and comments at the [Office Developer Platform UserVoice site](http://officespdev.uservoice.com/).
+You can use the experimental Desktop List View widget to display data in SharePoint lists. The widget displays data in read-only mode. Please provide ideas and comments at the Office Developer Platform UserVoice site.

```

Comment:
Microsoft no longer uses UserVoice... please correct the link: https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform